tests/backup/backup_delete_test.go

```diff
@@ -394,8 +396,38 @@ var _ = Describe("{DeleteIncrementalBackupsAndRecreateNew}", func() {
 				for _, vol := range bkpInspectResponse.GetBackup().GetVolumes() {
 					backupId := vol.GetBackupId()
 					log.InfoD(fmt.Sprintf("Backup Name: %s; BackupID: %s", incrementalBackupName, backupId))
-					dash.VerifyFatal(strings.Contains(backupId, "incr"), true,
-						fmt.Sprintf("Check if the backup %s is incremental or not ", incrementalBackupName))
+					if strings.Contains(backupId, "incr") {
+						dash.VerifyFatal(strings.Contains(backupId, "incr"), true,
+							fmt.Sprintf("Check if the backup %s is incremental or not ", incrementalBackupName))
+					} else {
+						// Attempting to take backups and checking if they are incremental or not
+						log.InfoD("New backup wasn't an incremental backup hence recreating new backup")
+						for i := 0; i < 3; i++ {
+							log.InfoD(fmt.Sprintf("Recreate incremental backup iternation: %d", i))
+							for _, namespace := range bkpNamespaces {
+								// Create a new incremental backups
+								incrementalBackupName = fmt.Sprintf("%s-%s-%v", "incremental-backup", namespace, time.Now().Unix())
+								incrementalBackupNamesRecreated = append(incrementalBackupNamesRecreated, incrementalBackupName)
```

Comment:
You can simply call this `incrementalBackupNames`

Reply:
Done